src/App.js

```diff
@@ -2,6 +2,10 @@ import Retool from "./components/Retool";
 import { useState } from "react";
 
 const App = () => {
+  const iframeStyle = {
```

Comment:
This example is getting a little ugly, not sure its worth including

Reply:
Happy either way, but I think more examples are better than fewer!